services/catalog/src/simcore_service_catalog/core/application.py

```diff
@@ -89,4 +90,7 @@ async def _add_process_time_header(request: Request, call_next: Callable):
             response.headers["X-Process-Time"] = str(process_time)
             return response
 
+    # gzip middleware
+    app.add_middleware(GZipMiddleware)
```

Comment:
so this zips if >500KB. How does help speed? is traffik already doing this?

Reply:
traefik does it only for the webserver. this is for internal only. and I think its if size>500bytes